react-backend/client/src/components/admin/adminVisitsPage.js

```diff
@@ -14,20 +14,48 @@ export class AdminVisitsPage extends Component {
     };
   }
 
+  headers = new Headers({
+    Authorization: localStorage.getItem('token'),
```

Comment:
Add a second header of `'Content-Type': 'application/json'` so we can use bodyparser with the request.

Reply:
it's so weird, but when I do that, the app does nothing - doesn't throw any errors and at the same time doesn't render visits component
when I remover this line - everything starts working again
what can be a problem here?

Reply:
everything is fine now :)